src/pages/contact-us/_contact-ways.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Text } from '../../components/elements/typography'
+import { Header } from 'components/elements'
+import { localize } from 'components/localization'
+import CallUsIcon from 'images/svg/call-us.svg'
+// import ChatLiveIcon from 'images/svg/chat-live.svg'
+import EmailUsIcon from 'images/svg/email-us.svg'
+import device from 'themes/device'
+
+const Wrapper = styled.section`
+    width: 100%;
+    height: 44.5rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: nowrap;
+    background-color: var(--color-white);
+    padding: 4rem 12.8rem;
+
+    @media ${device.tabletS} {
+        height: auto;
+    }
+`
+const WaysWrapper = styled.div`
+    display: flex;
+    flex-direction: row;
+
+    @media ${device.tabletS} {
+        flex-direction: column;
+        height: auto;
+    }
+`
+const Contact = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-wrap: nowrap;
+    width: 38.4rem;
+    padding-top: 3.2rem;
+`
+
+const StyledHeader = styled(Header)`
+    margin-bottom: 2.4rem;
+`
+const StyledText = styled(Text)`
+    margin-bottom: ${props => props.marginBttom || '0.8rem'};
+`
+const Logo = styled.div`
+    margin-bottom: 1.4rem;
+`
+const Splitter = styled.div`
+    background-color: #d8d8d8;
```

Comment:
can we use variable colors for this, check variables.js, and if we dont have this color, please add one 🙏